### PR TITLE
[CDAP-8396][CDAP-8554] Call to action for namespace creation

### PR DESCRIPTION
**Pull request by tonybach**
[CDAP-8396] https://issues.cask.co/browse/CDAP-8396
- Added CTA modal for namespace creation

[CDAP-8554] https://issues.cask.co/browse/CDAP-8554
- Fixed error where CTA modal was not fully overlapping with the creation wizard, when the view height is small

[CDAP-8412] https://issues.cask.co/browse/CDAP-8412
- This will also be resolved when this PR is merged.

### CODE REVIEW

File: cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js [hunk 1]
```diff
@@ -51,7 +55,27 @@ export default class AddNamespaceWizard extends Component {
       type: AddNamespaceActions.onReset
     });
   }
-
+  buildSuccessInfo(responseText) {
+    // responseText has the format "Namespace '{namespace}' created successfully.""
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js]: extra quote at the end.

Should open an improvement backend JIRA for this as well

Comment (tonybach) [reply cdap-ui/app/cdap/components/CaskWizards/AddNamespace/index.js]: Yep, created: https://issues.cask.co/browse/CDAP-8851.

**Comment by ajainarayanan** [comment]
@tonybach Any entity creation wizard's CTA has two links as per design however the "Go to Home page" link appears white in cdap and blue in pipelines and metadata views.
screen shot 2017-02-28 at 5 01 32 pm

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by tonybach** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC5962

**Pull request merged**